src/sentry/workflow_engine/process_update_helpers.py

```diff
@@ -0,0 +1,55 @@
+from sentry.incidents.utils.types import QuerySubscriptionUpdate
```

Comment:
can we move these methods back to `sentry/incidents`? that way we can keep all the metric alert code co-located and the workflow_engine stays as just the platform module.

Reply:
Sure, will do